manageiq-operator/deploy/crds/manageiq_v1alpha1_manageiq_cr.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: manageiq.example.com/v1alpha1
+kind: Manageiq
+metadata:
+  name: miq
+spec:
+  # Add fields here
+  appName:  "manageiq"
+  applicationAdminPassword: "smartvm" 
```

Comment:
Minor, but there is trailing whitespace in a number of locations in the various files (here is one example), if you could clean that up.